webpack/scenes/Subscriptions/components/SubscriptionsTable/components/Dialogs/index.js

```diff
@@ -17,10 +17,10 @@ const Dialogs = ({
 );
 
 Dialogs.propTypes = {
-  updateDialog: PropTypes.shape(UpdateDialog.propTypes).isRequired,
-  unsavedChangesDialog: PropTypes.shape(UnsavedChangesDialog.propTypes).isRequired,
-  inputsErrorsDialog: PropTypes.shape(InputsErrorsDialog.propTypes).isRequired,
-  deleteDialog: PropTypes.shape(DeleteDialog.propTypes).isRequired,
+  updateDialog: PropTypes.shape({}).isRequired,
+  unsavedChangesDialog: PropTypes.shape({}).isRequired,
+  inputsErrorsDialog: PropTypes.shape({}).isRequired,
+  deleteDialog: PropTypes.shape({}).isRequired,
```

Comment:
Are these necessary?  I would think they would all return objects so should be okay.

Reply:
This was failing tests as well

Reply:
:man_shrugging: